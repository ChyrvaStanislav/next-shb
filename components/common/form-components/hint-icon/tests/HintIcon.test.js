import React from 'react';
import { render } from '@testing-library/react';
import HintIcon from '../index';

const Component = () => <HintIcon />;

describe('HintIcon renders', () => {
  it('should HintIcon renders', () => {
    const { asFragment } = render(<Component />);
    expect(asFragment()).toMatchSnapshot();
  });
});
