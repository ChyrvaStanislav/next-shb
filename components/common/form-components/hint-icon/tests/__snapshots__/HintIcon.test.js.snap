// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HintIcon renders should HintIcon renders 1`] = `
<DocumentFragment>
  <section
    class="wrapperComponent"
    data-testid="popup"
    role="presentation"
    tabindex="1"
  >
    <div
      class="targetElement"
    >
      <div
        class="iconWrapper"
        data-testid="targetDiv"
      >
        <div
          class="icon"
        >
          i
        </div>
      </div>
    </div>
  </section>
</DocumentFragment>
`;
